Fix splash screen for iOS13 and below

ecotopian opened the pull request.

vauxhall reviewed (commented):
Why a xib and not a Storyboard?
Wasn't it the standard before SwiftUI to have a Storyboard as splash screen?

ecotopian commented:
> Why a xib and not a Storyboard?
> Wasn't it the standard before SwiftUI to have a Storyboard as splash screen?

afaik it does not make a difference - it was a .xib before, so I just reverted to orig. behaviour one and changed the color. Do you see any advantage in using a storyboard?

vauxhall commented:
> > Why a xib and not a Storyboard?
> > Wasn't it the standard before SwiftUI to have a Storyboard as splash screen?
> 
> @vauxhall: afaik it does not make a difference - it was a .xib before, so I just reverted to orig. behaviour one and changed the color. Do you see any advantage in using a storyboard?

If reverting from something that was already done is fine. For new developments I would not recommend using legacy components.
